internal/command/meta_ui.go

```diff
@@ -0,0 +1,53 @@
+package command
+
+import (
+	"github.com/mitchellh/cli"
+
+	"github.com/opentofu/opentofu/internal/command/views"
+)
+
+type WrappedUi struct {
```

Comment:
Could you add a comment explaining that this is a shim which adds json compatibility to commands that have not yet been refactored to support views?

Reply:
Added.